packages/assistant/src/RPA/Assistant/library.py

```diff
@@ -1081,7 +1089,7 @@ def add_next_ui_button(self, label: str, function: Union[Callable, str]):
 
         :param label: Text for the button
         :param function: Python function or Robot Keyword name, that will take form
-        results as it's first argument
+            results as it's first argument
```

Comment:
Nit: `results as its first argument`

Reply:
Fixes this in future PR at here https://github.com/robocorp/rpaframework/pull/830